### PR TITLE
[OAP-Cache-OAP] update plasma/external cache strategy readme

### PR DESCRIPTION
**Pull request by jikunshang**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by jikunshang** [comment]
@winningsix @zhixingheyi-tian  Please take a review, thanks!
I will update a Plasma readme in Arrow repo later.

File: oap-cache/oap/docs/OAP-User-Guide.md [hunk 1]
```diff
@@ -323,6 +337,30 @@ spark.sql.oap.cache.guardian.memory.size                   10g      # according
 ```
 Note: If "PendingFiber Size" (on spark web-UI OAP page) is large, or some tasks failed due to "cache guardian use too much memory", user could set `spark.sql.oap.cache.guardian.memory.size ` to a larger number, and the default size is 10GB. Besides, user could increase `spark.sql.oap.cache.guardian.free.thread.nums` or decrease `spark.sql.oap.cache.dispose.timeout.ms` to accelerate memory free.
 
+#### Use External cache strategy
+
+External cache strategy is implemented based on arrow/plasma library. To use this strategy, follow [prerequisites](#prerequisites-1) to set up DCPMM hardware and plasma library correctly, and start Plasma service on nodes, then refer below configurations to apply external cache strategy in your workload.
```
Comment (winningsix) [inline oap-cache/oap/docs/OAP-User-Guide.md:342]: DCPMM -> Optane PMem

File: oap-cache/oap/docs/OAP-User-Guide.md [hunk 2]
```diff
@@ -323,6 +337,30 @@ spark.sql.oap.cache.guardian.memory.size                   10g      # according
 ```
 Note: If "PendingFiber Size" (on spark web-UI OAP page) is large, or some tasks failed due to "cache guardian use too much memory", user could set `spark.sql.oap.cache.guardian.memory.size ` to a larger number, and the default size is 10GB. Besides, user could increase `spark.sql.oap.cache.guardian.free.thread.nums` or decrease `spark.sql.oap.cache.dispose.timeout.ms` to accelerate memory free.
 
+#### Use External cache strategy
```
Comment (winningsix) [inline oap-cache/oap/docs/OAP-User-Guide.md:340]: I am thinking we should have an ALL-In-One section here? Or at least pointing to users to the location of Plasma side document. Your thoughts?

Comment (jikunshang) [reply oap-cache/oap/docs/OAP-User-Guide.md:340]: Yes, we should have a external link about plasma details. We just need how to use plasma here.

**Comment by winningsix** [comment]
Let's do the rename in another PR.

**Pull request merged**